lib/options.json

```diff
@@ -321,6 +324,7 @@
       "filename": "should be {String|RegExp|Function} (https://webpack.js.org/configuration/dev-server/#devserver-filename-)",
       "port": "should be {String|Number} (https://webpack.js.org/configuration/dev-server/#devserver-port)",
       "socket": "should be {String} (https://webpack.js.org/configuration/dev-server/#devserver-socket)",
+      "sockPath": "should be {String} (https://webpack.js.org/configuration/dev-server/#devserver-sockPath)",
```

Comment:
This needs to be documented by opening a PR in [docs repo](https://github.com/webpack/webpack.js.org) please. Currently to URL redirects to the dev server 'landing page', which might be confusing for some and ultimately isn't really helpful